packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.stories.ts

```diff
@@ -134,3 +136,41 @@ export const WithCustomAppArea = {
     appArea: () => [h(OnyxIcon, { icon: placeholder, color: "secondary" }), "Custom name"],
   },
 } satisfies Story;
+
+/**
+ * This nav bar has a lot of menu and context area items.
+ * Both the nav area as well as the context area will overflow when opened.
+ */
+export const WithOverflowingMobileContent = {
+  args: {
+    ...WithContextArea.args,
+    mobileBreakpoint: ONYX_BREAKPOINTS.xl,
```

Comment:
```suggestion
    mobileBreakpoint: "xl",
```

The props also accepts just the breakpoint name :)